pysqldb3/query.py

```diff
@@ -557,7 +557,7 @@ def query_to_gpkg(dbo, query, path=None, gpkg_name=None, cmd=None, gpkg_tbl=None
                         gpkg_tbl = gpkg_tbl)
         
         if not gpkg_name:
```

Comment:
why not set as optional with default as ''?

Reply:
I tried this but it doesn't work. I have tests (`test_query_to_gpkg_basic_pth`) that don't pass if I default `gpkg_name = ''`. Even though I don't fill in the argument in the test, my tests fail due to `gpkg_name` defaulting to 'None'.